Common.xml

```diff
@@ -18,7 +18,7 @@
         <Submitter>OMA</Submitter>
         <!-- Description of the reusable resource -->
         <Description>
-          <![CDATA[The object link is used to refer an Instance of a given Object. An Object link value is composed of two concatenated 16-bits unsigned integers following the Network Byte Order convention. The Most Significant Halfword is an ObjectID, the Least Significant Hafword is an ObjectInstance ID.An Object Link referencing no Object Instance will contain the concatenation of 2 MAX-ID values (null link. Test 5]]>
+          <![CDATA[The object link is used to refer an Instance of a given Object. An Object link value is composed of two concatenated 16-bits unsigned integers following the Network Byte Order convention. The Most Significant Halfword is an ObjectID, the Least Significant Hafword is an ObjectInstance ID. An Object Link referencing no Object Instance will contain the concatenation of 2 MAX-ID values.]>
```

Comment:
A typo remains in this line: "the Least Significant Hafword", which should also be rectified

Reply:
Thanks Bill! Fixed now.